src/app.py

```diff
@@ -157,9 +179,68 @@ def appium_run(avd_name: str):
             cmd += ' --nodeconfig {file}'.format(file=CONFIG_FILE)
         except ValueError as v_err:
             logger.error(v_err)
+    elif grid4_connect == True:
+        local_ip = os.popen('ifconfig eth0 | grep \'inet\' | cut -d: -f2 | awk \'{ print $2}\'').read().strip()
+        try:
+            mobile_web_test = convert_str_to_bool(str(os.getenv('MOBILE_WEB_TEST', False)))
+            appium_host = os.getenv('APPIUM_HOST', local_ip)
+            appium_port = int(os.getenv('APPIUM_PORT', 4723))
+            plafform_name = 'android'
+            device_name = os.getenv('DEVICE', 'chrome')
+            browser_name = default_web_browser if mobile_web_test else device_name
+            create_node_config_selenium_grid_4(browser_name, appium_host, appium_port, plafform_name)
+            download_selenium_server_url = 'https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.7.0/selenium-server-4.7.1.jar'
```

Comment:
@budtmo @SHELA , I set it to version 4.7.1 since is the latest release that has the jar file from these assets at https://github.com/SeleniumHQ/selenium/releases/

Reply:
@LerryAlexander I see there's a `selenium-server-4.8.1.jar` available: https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.8.0/selenium-server-4.8.1.jar

Just need to expand the full `Assets` list under the latest release to see it: https://github.com/SeleniumHQ/selenium/releases/tag/selenium-4.8.0

Reply:
Thank you @esaari . 
Yeap, you are right, but the version here is not the major concern, on the discussion above with @budtmo we are trying to figure out a way to avoid downloading the jar file into the container and instead use some kind of an URL to access one fixed jar version.